docs/markdown/Python/python-integrations/protobuf-python.md

```diff
@@ -107,10 +107,24 @@ protobuf_sources(
 )
 ```
 
+> 📘 Use alternative grpc plugins
+>
+> By default, the `grpcio` plugin is used to generate the services stubs. Pants can also generate service stub for [grpclib](https://github.com/vmagamedov/grpclib).
+>
+> To use [grpclib](https://github.com/vmagamedov/grpclib), set `grpclib_plugin = true` in the `[python-protobuf]` scope. You probably also want to disable the `grpcio` plugin if you don't use it.
+>
+> ```toml
+> [python-protobuf]
+> grpclib_plugin = true
+> grpcio_plugin = false
+> ```
+>
+> Don't forget to add the `grpclib` or `grpclib[protobuf]` dependency to your requirements.
```

Comment:
It can't just be any version you add to your runtime requirements, it has to be the same one you built with, right? So we should explain how to find the default version Pants ships with, and point to https://www.pantsbuild.org/docs/python-lockfiles#sharing-lockfiles-between-tools-and-code to explain how to sync the two for custom versions (note that we point to other docs using the `doc:` directive, not a full URL, there are examples of this below).

Reply:
In fact, since we already discuss how to set up runtime requirements above in Step 2, I think these new docs belong there, and the explanation about how to sync runtime requirements versions with compile time requirements versions can go there, and be more general (since this is equally relevant to the `protobuf` and `grpcio` requirements, but we neglected to document that in the past...)

Reply:
I tried something, please let me know if that's what you had in mind.